cpp/ql/src/experimental/Security/CWE/CWE-415/DoubleFree.ql

```diff
@@ -0,0 +1,44 @@
+/**
+ * @name Errors When Double Free
+ * @description Double freeing of a previously allocated resource can lead to various vulnerabilities in the program
+ *              and requires the attention of the developer.
```

Comment:
I think the `requires the attention of the developer.` goes without saying. How about this:

```suggestion
 * @description Freeing a previously allocated resource twice can lead to various vulnerabilities in the program.
```